pkg/k8sutil/resource.go

```diff
@@ -14,6 +14,11 @@ import (
 	"k8s.io/client-go/dynamic"
```

Comment:
please remove that file and use `internal/source/kubernetes/k8sutil/resource.go` instead.

Reply:
As described above, I didn't wanted to depend our core to a plugin internal. I will be removing k8sutils anyways after platform migration, wdyt?